src/main/java/org/sciborgs1155/robot/drive/ModuleIO.java

```diff
@@ -43,6 +58,61 @@ public interface ModuleIO extends AutoCloseable, Logged {
   /** Resets all encoders. */
   void resetEncoders();
 
+  /**
+   * Returns the current state of the module.
+   *
+   * @return The current state of the module.
+   */
+  SwerveModuleState state();
```

Comment:
Pedantic but these getters should be moved to the top of each file, followed by updates and setters and other util methods